Source/AliveLib/Slurg.cpp

```diff
@@ -0,0 +1,29 @@
+#include "stdafx.h"
```

Comment:
This cpp and hpp needs adding to CMakeLists.txt otherwise it won't get compiled/linked